pg_search/src/api/mod.rs

```diff
@@ -54,6 +64,22 @@ impl AsInt for *mut pgrx::pg_sys::Node {
     }
 }
 
+#[cfg(not(not(any(feature = "pg13", feature = "pg14"))))]
+impl AsBool for *mut pgrx::pg_sys::Node {
+    unsafe fn as_bool(&self) -> Option<bool> {
+        let node = nodecast!(Value, T_Boolean, *self)?;
+        Some((*node).val.ival != 0)
+    }
+}
+
+#[cfg(not(any(feature = "pg13", feature = "pg14")))]
+impl AsBool for *mut pgrx::pg_sys::Node {
+    unsafe fn as_bool(&self) -> Option<bool> {
+        let node = nodecast!(Boolean, T_Boolean, *self)?;
+        Some((*node).boolval)
+    }
+}
```

Comment:
holy double negative batman! im not not not opposed to this if its necessary, I know its a common thing in JS, its it intentional here?

Reply:
haha, I f'd that up real good.  thanks!